storybook/src/elements/nuxeo-collapsible/nuxeo-collapsible.stories.js

```diff
@@ -0,0 +1,84 @@
+import { html } from 'lit-html';
+import { storiesOf } from '@storybook/polymer';
+import { boolean, text } from '@storybook/addon-knobs';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-collapsible';
+
+const stories = storiesOf('UI/nuxeo-collapsible', module);
+
+stories
+  .add('Default', () => {
+    const heading = text('Heading', 'Nuxeo');
+    const chevron = boolean('Chevron', true);
+    const chevronAtStart = boolean('Chevron at start', false);
+    const animation = boolean('Animation', true);
+    const expanded = boolean('Expanded', false);
```

Comment:
This is not in sync with the user interactions, it is pretty easy to break the synchronisation.
Not sure if we can do fix that, but just wanted to left the note if you want to try it out.

Reply:
We will keep this in mind to solve it later, with a more general approach to storybook since we may run into this situation more often